guides/doc-Provisioning_Guide/topics/proc_building-a-discovery-image.adoc

```diff
@@ -34,11 +34,18 @@ ifeval::["{build}" == "satellite"]
 ----
 endif::[]
 ifeval::["{build}" == "satellite"]
+
+* For the following {RHEL} 7 repositories required to build the Discovery image, change the download policy to *Immediate*. This is required because {Project} downloads all packages only during synchronization of repositories with the immediate download policy.
++
+** *Red Hat Enterprise Linux 7 Server Kickstart x86_64 7.7*
+** *Red Hat Satellite Capsule {ProductVersion} for RHEL 7 Server RPMs x86_64*
 +
-Synchronize the repositories that you want to use to build the Discovery image, for example:
+For more information about changing download policies, see {baseurl}content_management_guide/importing_red_hat_content#changing_the_download_policy_for_a_repository[Changing the Download Policy for a Repository] in the _Content Management Guide_.
+
+* Synchronize the following {RHEL} 7 repositories required to build the Discovery image:
 +
-* *Red Hat Enterprise Linux 7 Server Kickstart x86_64 7.7*.
-* *Red Hat Satellite Capsule {ProductVersion} for RHEL 7 Server RPMs x86_64*.
+** *Red Hat Enterprise Linux 7 Server Kickstart x86_64 7.7*
```

Comment:
All good but one nitpick, this should be updated to 7.8 (latest stable release). Can you introduce new variable called something like `fdi-el-version` and find occurances of 7.7 in this file and replace it with the variable? Set it to 7.8 for now.

Reply:
This will require manual maintenance of this attribute every time a new minor RHEL7 is released. I would rather say that you need the latest y-stream version of RHEL 7. For example, 7.8.

Reply:
I actually see the kickstart repository for 7.9 in the web UI now.